x-pack/test_serverless/functional/test_suites/observability/navigation.ts

```diff
@@ -0,0 +1,90 @@
+    before(async () => {
+      await svlObltNavigation.navigateToLandingPage();
+    });
+
+    it('navigate observability sidenav & breadcrumbs', async () => {
+      const expectNoPageReload = await svlCommonNavigation.createNoPageReloadCheck();
+
+      // check serverless search side nav exists
+      await svlCommonNavigation.expectExists();
+      await svlCommonNavigation.breadcrumbs.expectExists();
+      await svlObltOnboardingPage.assertQuickstartBadgeExists();
+
+      // check side nav links
+      await svlCommonNavigation.sidenav.expectSectionOpen('observability_project_nav');
+      await svlCommonNavigation.sidenav.expectLinkActive({ deepLinkId: 'observabilityOnboarding' });
+      await svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({
+        deepLinkId: 'observabilityOnboarding',
+      });
+      await svlCommonNavigation.sidenav.expectSectionClosed('project_settings_project_nav');
+
+      // TODO: test something oblt project specific instead of generic discover
+      // navigate to discover
+      await svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'discover' });
+      await svlCommonNavigation.sidenav.expectLinkActive({ deepLinkId: 'discover' });
+      await svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({ deepLinkId: 'discover' });
+      await expect(await browser.getCurrentUrl()).contain('/app/discover');
+
+      // check the aiops subsection
+      await svlCommonNavigation.sidenav.clickLink({ navId: 'aiops' }); // open ai ops subsection
+      await svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'ml:anomalyDetection' });
+      await svlCommonNavigation.sidenav.expectLinkActive({ deepLinkId: 'ml:anomalyDetection' });
+      await svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({ text: 'AIOps' });
+      await svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({
+        deepLinkId: 'ml:anomalyDetection',
+      });
+
+      // navigate to a different section
+      await svlCommonNavigation.sidenav.openSection('project_settings_project_nav');
+      await svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'management' });
+      await svlCommonNavigation.sidenav.expectLinkActive({ deepLinkId: 'management' });
+      await svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({ deepLinkId: 'management' });
+
+      // navigate back to serverless oblt overview
+      await svlCommonNavigation.breadcrumbs.clickHome();
+      await svlCommonNavigation.sidenav.expectLinkActive({ deepLinkId: 'observabilityOnboarding' });
+      await svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({
+        deepLinkId: 'observabilityOnboarding',
+      });
+      await svlCommonNavigation.sidenav.expectSectionOpen(`project_settings_project_nav`); // remains open
+
+      await expectNoPageReload();
+    });
+
+    // TODO: bug in the management nav config? @sebelga is looking into it
+    it.skip('active sidenav section is auto opened on load', async () => {
```

Comment:
I think we can unskip it now as it was fixed in https://github.com/elastic/kibana/pull/161773

Reply:
I just tried, seems like it doesn't work
run observability, open "project settings > management", reload the page, see that project settings page is not collapsed.  

I think you said that this is the reason: https://github.com/elastic/kibana/blob/87a9577821341e57ee349c990140892e6da28317/x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx#L133

but I am not sure what this option mean exactly and if we can just change it to "false"

Reply:
I unskipped and removed the flag, I think it makes sense. https://github.com/elastic/kibana/pull/161856/commits/ba38f695b8f8edb3d0e8f4ca0a4fd3d3c3f2dc7c

Reply:
Yes we need to remove that flag. I did it in https://github.com/elastic/kibana/pull/161971 (not yet merged).
`defaultIsCollpased` forces the state to be collapse by default. Leaving it `undefined` allows us to control it through active state (I moved it to the top of the options in the doc: https://github.com/elastic/kibana/pull/161971/files#diff-2ce754a68c80c33e2a0af4666d8b7fcd9d539bdea322fddfd169c7ff7d560b71R97)